tests/smi_count/smi_count.cpp

```diff
@@ -40,11 +41,11 @@ static int smi_count_run(struct test *test, int cpu)
     return EXIT_SUCCESS;
 }
 
-
 DECLARE_TEST(smi_count, "Counts SMI events")
+    .test_init = [](struct test *t) { return InterruptMonitor::InterruptMonitorWorks ? EXIT_SUCCESS : EXIT_SKIP; },
     .test_run = smi_count_run,
+    .test_cleanup = [](struct test * t) { initialize_smi_counts(); return EXIT_SUCCESS; },
```

Comment:
This runs in the child process, so its effects will be discarded. Might as well not do anything.

Reply:
Yes, I was conscious of that, but I wanted to have it there for potentially changing this to no-fork mode (and then it will be important to reset it here).